### DISCUSSION TITLE
Cannot start nuxt:  Named export 'rollup' not found.

### DISCUSSION THREAD

**Discussion by k22pr** [nuxt/nuxt#18257] [Questions] [answered]
It was installed according to the official document of nuxt3, but yarn dev does not work normally.
However, an error called ' ERROR  Cannot start nuxt:  Named export 'rollup' not found. The requested module 'rollup' is a CommonJS module, which may not support all module.exports as named exports.                                                            14:55:21
CommonJS modules can always be imported via the default export, for example using:' is output.

I installed it with that code.
```
npx nuxi init nuxt3-app
```

and running code
```
yarn dev (nuxi dev)
```

<img width="1050" alt="스크린샷 2021-10-25 오후 2 50 26" src="https://user-images.githubusercontent.com/12038868/138641276-e97a6df0-6915-47af-91d1-780dc40d3ffc.png">

node version
```
node --version
v14.16.1
```

Did I miss anything?

**Comment by wobsoriano** [comment, marked as answer]
https://github.com/nuxt/framework/issues/1428
